v2 volume: change default hugepage to 2048 MiB (#837)
longhorn/website: content/docs/1.6.0/references/settings.md

>
> - When the V2 Data Engine is enabled, each instance-manager pod utilizes 1 CPU core. This high CPU usage is attributed to the spdk_tgt process running within each instance-manager pod. The spdk_tgt process is responsible for handling input/output (IO) operations and requires intensive polling. As a result, it consumes 100% of a dedicated CPU core to efficiently manage and process the IO requests, ensuring optimal performance and responsiveness for storage operations.

#### V2 Data Engine HugePage Limit

> Default: `2048`

> This allows users to configure HugePage size in MiB for v2 data engine.

Reviewer: ```suggestion
Maximum huge page size (in MiB) for the V2 Data Engine.
```
